### PR TITLE
Update openwrt version to reflect changes in etherwake-nfqueue

### PR DESCRIPTION
**Pull request by 7ritn**
This PR updates the OpenWRT repository of etherwake-nfqueue to the current version of etherwake-nfqueue

### CODE REVIEW

File: net/etherwake-nfqueue/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
+PKG_RELEASE:=3
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/mister-benjamin/etherwake-nfqueue.git
-PKG_SOURCE_DATE:=2019-09-28
-PKG_SOURCE_VERSION:=f71c269b58585e93575fa3e9fcc1793806fb3080
-PKG_MIRROR_HASH:=4960dc592abc4ca06504c92ca09fc736c678353df0dcc32d4081e17b137a9164
+PKG_SOURCE_DATE:=2023-06-20
+PKG_SOURCE_VERSION:=857c8714c4aec2b54ae70655c6d0e13f3c70ed0a
+PKG_MIRROR_HASH:=skip
```
Comment (mister-benjamin) [inline net/etherwake-nfqueue/Makefile]: Could you generate and fill in the hash?

Comment (7ritn) [reply net/etherwake-nfqueue/Makefile]: Oh sorry I did that already but I think I forgot to commit it. Will do

**Comment by mister-benjamin** [comment]
Could you squash the commits?

File: net/etherwake-nfqueue/files/etherwake-nfqueue.config [hunk 2]
```diff
@@ -14,6 +14,9 @@ config etherwake-nfqueue 'setup'
 # # uci set etherwake-nfqueue.@target[-1].interface=eth0
 # Configure if it should be sent to broadcast address, defaults to off
 # # uci set etherwake-nfqueue.@target[-1].broadcast=off
+# Optionally enable defer delivery mechanism and provide an IP-Address of the host to be woken up
```
Comment (mister-benjamin) [inline net/etherwake-nfqueue/files/etherwake-nfqueue.config]: Can you remind me about which host's IP address it is again? IIRC, it's not the one to be woken up?

File: README.md [hunk 3]
```diff
@@ -146,6 +146,7 @@ The full list of options for a target is:
 | interface   | no       | The interface used for sending the magic packet, default is interface=eth0 |
 | broadcast   | no       | Send magic packet to broadcast address, default is broadcast=off |
 | password    | no       | Set a password (required by some adapters), e.g. password=00:22:44:66:88:aa or 192.168.1.1 |
+| defer       | no       | Defer delivery of matched packets until host with specified IP-Adress responds to a ping i.e. has woken up |
```
Comment (mister-benjamin) [inline README.md]: Typo in "Adress". Also I think "IP address" is more common than "IP-Address" - there's one more occurrence below.

File: net/etherwake-nfqueue/files/etherwake-nfqueue.config (hunk 2 above)
Comment (7ritn) [reply net/etherwake-nfqueue/files/etherwake-nfqueue.config]: Since magic packets are send to MAC adresses, but the ping works on IPs so an IP is required

Comment (mister-benjamin) [reply net/etherwake-nfqueue/files/etherwake-nfqueue.config]: I mean I forgot what this defer mechanism was about again - it was confusing me at the time we merged it, and now I forgot it again :-) The way I remember it (or initially understood it) is that the IP address given here is pinged and only if it's online, the magic packet is sent to the specified MAC address - how else would a magic packet ever be sent when the host to be woken up as to be online in order to be woken up. But I guess I just forgot how this feature is supposed to be used.

Comment (7ritn) [reply net/etherwake-nfqueue/files/etherwake-nfqueue.config]: The idea is that packet don't get lost but are send to the host when it has woken up. This helps protocols without resending mechanism, but also should prevent ICMP unreachable replies, where for example TCP could give up.

Comment (mister-benjamin) [reply net/etherwake-nfqueue/files/etherwake-nfqueue.config]: Ah, I see: It's not about deferring the magic packets, but buffering any TCP/UDP packets addressed to the sleeping host, right?

Comment (7ritn) [reply net/etherwake-nfqueue/files/etherwake-nfqueue.config]: Yeah exactly

File: README.md (hunk 3 above)
Comment (7ritn) [reply README.md]: Fixed

**Comment by 7ritn** [comment]
Please squash and merge, so commit history is kept :)

**Comment by mister-benjamin** [comment]
Done. Thanks.

Will you open a pull request in https://github.com/openwrt/packages, or do you want me to do it?

**Comment by 7ritn** [comment]
Sure I can do that, however I'm somewhat confused since the Makefile here and in the packages repository have different dependencies (+iptables-mod-nfqueue) vs (+nftables +kmod-nft-queue). Was this repository simply not updated to reflect the changes?

Plus I think I also need to add the libmnl as a dependecy since the project now depends on it, right?

**Comment by mister-benjamin** [comment]
I will take a look later today.

**Comment by mister-benjamin** [comment]
@7ritn Sorry for the delay. The dependencies of the OpenWrt package were updated with https://github.com/openwrt/packages/commit/5e894cae13d2b6956412f7dd7da36fba752a55c1, because OpenWrt switched from iptables to nft some time ago. The iptables dependency was listed only because of how the rules were configured in user space, and the corresponding kernel module package is now kmod-nft-queue. If you would like to update the package in OpenWrt, I suggest you build and test it with the dependencies currently listed upstream, and open a pull request. Unfortunately I currently don't have the time to do this myself.

**Pull request merged**